torch/linalg/__init__.py

```diff
@@ -8,6 +8,76 @@
 # Note: This not only adds doc strings for functions in the linalg namespace, but
 # also connects the torch.linalg Python namespace to the torch._C._linalg builtins.
 
+cholesky = _add_docstr(_linalg.linalg_cholesky, r"""
+linalg.cholesky(input, *, out=None) -> Tensor
+
+Computes the Cholesky decomposition of a Hermitian (or symmetric for real-valued matrices)
+positive-definite matrix or the Cholesky decompositions for a batch of such matrices.
+The returned matrix ``L`` is lower-triangular, and
+the decomposition has the form:
+
+.. math::
+
+    \text{input} = LL^H
+
+where :math:`L^H` is the conjugate transpose of :math:`L`, which is just a transpose for the case
+of real-valued input matrices.
+In code it translates to ``input = L @ L.t()` if :attr:`input` is real-valued and
+``input = L @ L.conj().t()`` if :attr:`input` is complex-valued.
```

Comment:
Then, following the above change, the return value has not yet been mentioned, so a sentence can be added here:

"The :math:`L` matrix from each decomposition is returned."